pkg/restore/restic_restore_action.go

```diff
@@ -123,3 +130,72 @@ func (a *ResticRestoreAction) Execute(input *velero.RestoreItemActionExecuteInpu
 
 	return velero.NewRestoreItemActionExecuteOutput(&unstructured.Unstructured{Object: res}), nil
 }
+
+func getImage(log logrus.FieldLogger, config *corev1.ConfigMap) string {
+	if config == nil {
+		log.Debug("No config found for plugin")
+		return initContainerImage(defaultImageBase)
+	}
+
+	image := config.Data["image"]
+	if image == "" {
+		log.Debugf("No custom image configured")
+		return initContainerImage(defaultImageBase)
+	}
+
+	log = log.WithField("image", image)
+
+	parts := strings.Split(image, ":")
+	switch {
+	case len(parts) == 1:
+		// tag-less image name: add tag
+		log.Debugf("Plugin config contains image name without tag")
```

Comment:
Would this be more useful: `s/Plugin config contains image name without tag/Plugin config contains image name without tag. Adding tag.`

Reply:
wfm